app/src/main/java/com/cornellappdev/android/eatery/LogoutFragment.java

```diff
@@ -0,0 +1,105 @@
+package com.cornellappdev.android.eatery;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
+import android.widget.RelativeLayout;
+
+import com.cornellappdev.android.eatery.presenter.AccountPresenter;
+
+public class LogoutFragment extends Fragment {
+
+    private Button mLogoutButton;
+    private CheckBox mSaveInfoCheck;
+    private RelativeLayout mAboutButton;
```

Comment:
should this be a RelativeLayout? or what is this??

Reply:
Yeah my naming was bad I changed it to mAboutArea. I added the onclicklistener to the entire relativelayout so people could click anywhere within the tab to renavigate